optaplanner-bom/pom.xml

```diff
@@ -0,0 +1,487 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <!-- TODO In step 3, this must extend optaplanner-parent instead and stop duplicating this metadata -->
+    <!-- But for that to work, optaplanner-parent must have an empty dependencyManagement (so no kie-parent parent) -->
+    <groupId>org.kie</groupId>
+    <artifactId>kie-user-bom-parent</artifactId>
+    <version>7.40.0-SNAPSHOT</version>
+  </parent>
```

Comment:
Should have empty <relativePath/> otherwise:

[WARNING] 'parent.relativePath' of POM org.optaplanner:optaplanner-bom:8.0.0-SNAPSHOT (/home/jlocker/src/github.com/kiegroup/optaplanner/optaplanner-bom/pom.xml) points at org.optaplanner:optaplanner-parent instead of org.kie:kie-user-bom-parent, please verify your project structure @ line 6, column 11

Reply:
Good point, fixed :)